index.js

```diff
@@ -0,0 +1,108 @@
+let calculateBtn = document.getElementsByTagName('button')[0];
+
+let resultMessage = document.querySelector(' div:nth-child(6) > article > ul > li:nth-child(4) > div > div > ul > li:nth-child(3) > div');
+
+let errorMessage = document.querySelector('div:nth-child(5) > article > ul > li:nth-child(4) > div > div > ul > li:nth-child(3) > div')
+
+let result;
+
+
+calculateBtn.addEventListener('click', function () {
+    
+    // logic to erase the errors before any other calculations
+    errorMessage.innerHTML = '';
+    resultMessage.innerHTML = '';
+
+
+    // logic to select and manipulate the input fields and dropdowns
+    let valueInputOne = document.getElementById('firstOperand').value;
+    let valueInputTwo = document.getElementById('secondOperand').value;
+    let valueInputThree = document.getElementById('thirdOperand').value;
+    let valueInputFour = document.getElementById('fourthOperand').value;
+
+    let selectOne = document.getElementsByName('operations')[0].value;
+    let selectTwo = document.getElementsByName('operations')[1].value;
+    let selectThree = document.getElementsByName('operations')[2].value;
+
+
+    getErrorMessageOnOperands(valueInputOne, valueInputTwo, selectOne);
+    if (getErrorMessageOnOperands(valueInputOne, valueInputTwo, selectOne)) {
```

Comment:
it does not look necessary to call getErrorMessageOnOperands twice since it is called in the if statement also